core/src/raw/ops.rs

```diff
@@ -124,6 +124,12 @@ impl OpList {
         self.start_after.as_deref()
     }
 
+    /// Set the start_after of list operation.
+    pub fn set_start_after(mut self, start_after: &Option<String>) -> Self {
```

Comment:
Hi, please don't add new API to raw::ops in this PR.

Reply:
Thanks for the reminder, fixed.